src/main/java/mate/academy/util/ConnectionUtil.java

```diff
@@ -0,0 +1,28 @@
+package mate.academy.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql:"
```

Comment:
```suggestion
    private static final String DB_URL = "jdbc:mysql://localhost:3306/mydatabase?serverTimezone=UTC";
```
any concatenation creating new String object.